logic/build.gradle.kts

```diff
@@ -75,6 +75,9 @@ kotlin {
         val androidMain by getting {
             dependencies {
                 implementation(Dependencies.Android.work)
+                implementation(Dependencies.Firebase.firebaseBOM)
+                implementation(Dependencies.Firebase.firebaseCloudMessaging)
+                implementation(Dependencies.Firebase.abcNotifications)
```

Comment:
Are we using this?

Can we remove it?

I don't think we may nor should introduce proprietary libraries in Kalium. If we want to make a F-droid build for the app later, Kalium should be completely free of this.